Lib/unittest/case.py

```diff
@@ -339,16 +339,17 @@ def __exit__(self, exc_type, exc_value, tb):
                 .format(logging.getLevelName(self.level), self.logger.name))
 
 
-def _short_description_from_docstring(text):
-    # Return the summary line from a docstring.
-    # If there is no summary line, return None.
-    if text:
-        (synopsis, long_description) = pydoc.splitdoc(text)
-        synopsis = synopsis.strip()
-    else:
-        # The text is either an empty string, or some other false value.
-        synopsis = None
+def _get_short_description(doc):
+    """
+    Return the summary line from a docstring.
+
+    If there is no summary line, return None.
```

Comment:
> If there is no summary line, return None.

I don't think this is accurate. If there is no summary line, it can also return the empty string.